blob/driver/driver.go

```diff
@@ -100,6 +100,12 @@ type WriterOptions struct {
 	// Metadata holds key/value strings to be associated with the blob.
 	// Keys are guaranteed to be non-empty and lowercased.
 	Metadata map[string]string
+	// When true, if ContentType is the empty string, it will stay the empty
```

Comment:
This comment is for driver implementors, so copying what's in blob.go doesn't quite work. Something like:

When true, the driver should attempt to disable any automatic content-type detection that the provider applies on writes with an empty ContentType.

Reply:
Done, thanks.